docs/services/sharelatex.md

```diff
@@ -13,28 +13,28 @@ ShareLaTeX requires a redis and a mongodb instance!
     volumes:
       - /srv/storage/paper/data:/var/lib/sharelatex
     environment:
-      - SHARELATEX_APP_NAME: ShareLaTeX
-      - SHARELATEX_MONGO_URL: mongodb://mongo/sharelatex
-      - SHARELATEX_REDIS_HOST: redis
-      - REDIS_HOST: redis
-      - ENABLED_LINKED_FILE_TYPES: 'url,project_file'
-      - ENABLE_CONVERSIONS: 'true'
-      - EMAIL_CONFIRMATION_DISABLED: 'true'
-      - TEXMFVAR: /var/lib/sharelatex/tmp/texmf-var
-      - SHARELATEX_SITE_URL: https://paper.domain.de
-      - SHARELATEX_NAV_TITLE: ShareLaTeX
-      - SHARELATEX_LEFT_FOOTER: '[]'
-      - SHARELATEX_RIGHT_FOOTER: '[]'
-      #- SHARELATEX_HEADER_IMAGE_URL: http://somewhere.com/mylogo.png
-      #- SHARELATEX_EMAIL_FROM_ADDRESS: "team@sharelatex.com"
-      #- SHARELATEX_EMAIL_SMTP_HOST: smtp.mydomain.com
-      #- SHARELATEX_EMAIL_SMTP_PORT: 587
-      #- SHARELATEX_EMAIL_SMTP_SECURE: false
-      #- SHARELATEX_EMAIL_SMTP_USER:
-      #- SHARELATEX_EMAIL_SMTP_PASS:
-      #- SHARELATEX_EMAIL_SMTP_TLS_REJECT_UNAUTH: true
-      #- SHARELATEX_EMAIL_SMTP_IGNORE_TLS: false
-      #- SHARELATEX_CUSTOM_EMAIL_FOOTER: "This system is run by department x"
+      - SHARELATEX_APP_NAME=ShareLaTeX
+      - SHARELATEX_MONGO_URL=mongodb://mongo/sharelatex
+      - SHARELATEX_REDIS_HOST=redis
+      - REDIS_HOST=redis
+      - ENABLED_LINKED_FILE_TYPES='url,project_file'
+      - ENABLE_CONVERSIONS='true'
+      - EMAIL_CONFIRMATION_DISABLED='true'
+      - TEXMFVAR=/var/lib/sharelatex/tmp/texmf-var
+      - SHARELATEX_SITE_URL=https://paper.domain.de
+      - SHARELATEX_NAV_TITLE=ShareLaTeX
+      - SHARELATEX_LEFT_FOOTER='[]'
+      - SHARELATEX_RIGHT_FOOTER='[]'
+      #- SHARELATEX_HEADER_IMAGE_URL=http://somewhere.com/mylogo.png
+      #- SHARELATEX_EMAIL_FROM_ADDRESS="team@sharelatex.com"
+      #- SHARELATEX_EMAIL_SMTP_HOST=smtp.mydomain.com
+      #- SHARELATEX_EMAIL_SMTP_PORT=587
+      #- SHARELATEX_EMAIL_SMTP_SECURE=false
+      #- SHARELATEX_EMAIL_SMTP_USER=
+      #- SHARELATEX_EMAIL_SMTP_PASS=
+      #- SHARELATEX_EMAIL_SMTP_TLS_REJECT_UNAUTH=true
+      #- SHARELATEX_EMAIL_SMTP_IGNORE_TLS=false
+      #- SHARELATEX_CUSTOM_EMAIL_FOOTER="This system is run by department x"
```

Comment:
add Quotation mark arround the whole environment variable, not only arround the value

e.g.
```yaml
labels:
  - "SHARELATEX_CUSTOM_EMAIL_FOOTER=This system is run by department x"
```